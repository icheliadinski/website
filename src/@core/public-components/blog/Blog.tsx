import React from 'react'
import { SideCard } from '@core/ui-kit'

export const Blog = () => {
  return (
    <SideCard>
      <article>My blog</article>
    </SideCard>
  )
}
