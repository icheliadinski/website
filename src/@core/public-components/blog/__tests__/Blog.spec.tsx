import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { Blog } from '../Blog'

describe('[Blog]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<Blog />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
