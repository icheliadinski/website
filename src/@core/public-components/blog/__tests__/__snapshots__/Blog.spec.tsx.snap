// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Blog] renders successfully 1`] = `
<SideCard>
  <article>
    My blog
  </article>
</SideCard>
`;
